cgp/individual.py

```diff
@@ -171,11 +171,11 @@ def _update_parameters_from_numpy_array(
         return genome.update_parameters_from_numpy_array(params, params_names)
 
     def __lt__(self, other: "IndividualBase") -> bool:
-        for i in range(len(self._fitness)):
+        for i in range(len(self._fitness) - 1, -1, -1):
```

Comment:
If I understand this correctly this makes it loop backwards (which is correct I think), but why start at `len()-1` and end at `-1` instead of `len` and end at `0`?

Reply:
Ah I guess it is to access the index below. However maybe this should be commented or documented somehow?(personally I had too look up the python documentation to make sure I understand the logic correctly :see_no_evil:

Reply:
yes, this was a bit too complicated :D simplified it now